requirements.txt

```diff
@@ -14,3 +14,5 @@ pymessagebus~=1.2
 click~=8.1.0
 openpyxl==3.0.9
 colorama~=0.4
+fastapi~=0.77
+uvicorn[standard]
```

Comment:
uvicorn, might have to be placed in another requirements.txt. 
I hope fresh environments are still installable with  pip, and for airflow.